lib/pages/chat_list/chat_list_item.dart

```diff
@@ -201,140 +205,189 @@ class ChatListItem extends StatelessWidget {
+                    width: isMentionned ? ChatListItemStyle.mentionIconWidth : 0,
                     decoration: BoxDecoration(
-                      color: room.highlightCount > 0 ||
-                              room.membership == Membership.invite
-                          ? Colors.red
-                          : room.notificationCount > 0 || room.markedUnread
-                              ? Theme.of(context).colorScheme.primary
-                              : Theme.of(context).colorScheme.primaryContainer,
-                      borderRadius:
-                          BorderRadius.circular(AppConfig.borderRadius),
+                      color: Theme.of(context).colorScheme.primary,
+                      borderRadius: BorderRadius.circular(AppConfig.borderRadius),
                     ),
                     child: Center(
-                      child: room.notificationCount > 0
+                      child: isMentionned
                           ? Text(
-                              room.notificationCount.toString(),
-                              style: TextStyle(
-                                color: room.highlightCount > 0
-                                    ? Colors.white
-                                    : room.notificationCount > 0
-                                        ? Theme.of(context)
-                                            .colorScheme
-                                            .onPrimary
-                                        : Theme.of(context)
-                                            .colorScheme
-                                            .onPrimaryContainer,
-                                fontSize: 13,
-                              ),
-                            )
+                                  '@',
+                                  style: TextStyle(
+                                    color: isMentionned
+                                        ? Theme.of(context).colorScheme.onPrimary
+                                        : Theme.of(context).colorScheme.onPrimaryContainer,
+                                    fontSize: 12,
+                                  ),
+                                )
                           : Container(),
                     ),
-                  ),
-                ],
+                  );
+                },
               ),
-              onTap: () => clickAction(context),
-            ),
-          ],
+              const SizedBox(width: 4),
+              AnimatedContainer(
+                duration: FluffyThemes.animationDuration,
+                curve: FluffyThemes.animationCurve,
+                padding: const EdgeInsets.symmetric(horizontal: 7),
+                height: unreadBubbleSize,
+                width: room.notificationCount == 0 && !unread && !room.hasNewMessages
+                    ? 0
+                    : (unreadBubbleSize - 9) * room.notificationCount.toString().length + 9,
```

Comment:
What is 9 ? IMO, you should make a const for this number

Reply:
dont use the magic number